2021q3/linuxulator.adoc

```diff
@@ -0,0 +1,24 @@
+=== Linux compatibility layer update
+
+Contact: Dmitry Chagin, <dchagin@FreeBSD.org> +
+Contact: Edward Tomasz Napierala, <trasz@FreeBSD.org>
+
+The goal of this project is to improve FreeBSD's ability to execute unmodified Linux binaries.
+Current support status of specific Linux applications is being tracked at the link:https://wiki.freebsd.org/LinuxApps[Linux app status Wiki page].
```

Comment:
If any applications are newly working under Linuxulator might like to highlight them? But major improvements here are probably in the previous qtr?

Reply:
I'm not aware of any major ones.  To me the biggest change from the user point of view is that Linuxulator is now "generally usable" on arm64.